lib/gui/app/pages/main/MainPage.tsx

```diff
@@ -87,9 +88,7 @@ const StepBorder = styled.div<{
 	position: relative;
 	height: 2px;
 	background-color: ${(props) =>
-		props.disabled
-			? props.theme.colors.dark.disabled.foreground
-			: props.theme.colors.dark.foreground};
+		props.disabled ? colors.dark.disabled.foreground : colors.dark.foreground};
```

Comment:
I had some typing issues when I first made the branch when updating some modules and I had to change the theme as well, I'll revert this since I tested it and it now works again